src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1962,29 +1965,45 @@ private void printDevModeMessages(boolean inputUnavailable, boolean startup) {
         }
         if (startup) {
             if (container) {
+                boolean nonDefaultHttpPortUsed = !skipDefaultPorts && !String.valueOf(LIBERTY_DEFAULT_HTTP_PORT).equals(httpPort);
+                boolean nonDefaultHttpsPortUsed = !skipDefaultPorts && !String.valueOf(LIBERTY_DEFAULT_HTTPS_PORT).equals(httpsPort);
+                int debugPort = (alternativeDebugPort == -1 ? libertyDebugPort : alternativeDebugPort);
+                boolean nonDefaultDebugPortUsed = !skipDefaultPorts && LIBERTY_DEFAULT_DEBUG_PORT != debugPort;
```

Comment:
`skipDefaultPorts` should not affect this boolean, since it is not related to the debug port.  Perhaps it should check if `libertyDebug` is true instead .

Reply:
```
[INFO] *    Liberty container port information:
[WARNING] *        The Liberty container is using non-default host ports.
[INFO] *        Internal container HTTP port [ 9080 ] is mapped to Docker host port [ 9080 ]
[INFO] *        Internal container HTTPS port [ 9443 ] is mapped to Docker host port [ 9443 ]
[INFO] *        Liberty debug port mapped to Docker host port: [ 60141 ] <
[INFO] ************************************************************************
```